lib/builderator/control/version.rb

```diff
@@ -32,6 +32,12 @@ class Version
       class << self
         def current
           @current ||= SCM.tags.last
+
+          if @current.nil? && Util.relative_path('VERSION').exist?
+            @current ||= Version.from_string(Util.relative_path('VERSION').read)
+          end
+
+          @current
```

Comment:
@erran-r7 @athompson-r7 I like the idea of an error message.

I avoided picking a default version number because I'm not sure _what_ to pick. I've seen projects use 0.0.1, 0.1.0, and 1.0.0, all with valid reasons for why that's the right default.